### DISCUSSION TITLE
Eclipse Station

### DISCUSSION THREAD

**Discussion by MirceaKitsune** [redeclipse/base#1443] [Maps and Content Showcase]
This started out as an attempt to create my own spaceship map inspired by UT's Hyperblast: The initial design was a failure with cramped corridors full of things you'd hit your head against. I decided to do a deeper redesign and turn it into a sort of Earth orbiting station, basing some of how I remembered other space maps such as Phobos being. I managed to put together an unique navigable design, which upon a simple test seems to play very well. The map is in an early stage at the time of posting, it's intended for Vanilla and will hopefully make it in time for 2.1.0.

- [station_r1600.zip](https://github.com/redeclipse/base/files/12775995/station_r1600.zip)

First version of the map. Contains the finished layout with basic detail for testing, much more detailing is needed. Everything essential is done except for bot waypoints, it can still be explored and tested with bots but they won't more around, the next version should have navigation clarified so that waypoints may be included. Includes affinity entities, but note this map is not ideal for CTF or BB due to how small it is and easy to get between the two bases, all modes are however supported.

- [station_r2036.zip](https://github.com/redeclipse/base/files/12797926/station_r2036.zip)

Added lots of detail, minor architecture changes, optimized lights and particles for performance. The exterior jump pad was disabled, rocket is now located at the bottom in a small hidden area which requires a risky jump! Bot waypoints done for all main areas without jumps just yet: I'm hoping the developers can implement a low gravity entity so I can set low gravity in the outside area only, so far everything is normal gravity for sanity's sake. I'd say the map is about 90% / 95% done at this stage.

- [station_r2199.zip](https://github.com/redeclipse/base/files/13059289/station_r2199.zip)

Lots of light and performance optimizations as well as texture fixes, minor geometry changes and extra detail. Map is now called Eclipse Station. Culled textures for better loading times. I consider the map ready to be included in Git and tested.

- [station_r2341.zip](https://github.com/redeclipse/base/files/13064981/station_r2341.zip)

Further details and misc fixes. Fix teleporter bug pointed out by ServerMom on Discord, a thruster was added behind it with some larger models detailing the lower area. Use `lightfx` for detail lights and particles. Give names to control points in DAC. Added bot waypoints for jumping and falling, may need to be redone when low gravity can be properly implemented.

- [station_r2439.zip](https://github.com/redeclipse/base/files/13079893/station_r2439.zip)

A few more hopefully final fixes. Decreased the radius of most lights and disabled some light volumes to further improve performance. New window in the tower.

- [station_r2579.zip](https://github.com/redeclipse/base/files/13401355/station_r2579.zip)

Final improvements following feedback from DC and Jigoku. Contains fixes for engine related collision issues, more texture alignment and usage improvements, as well as fixing some places the player could still get stuck in or annoyances for wall running. In addition the large round window in the observatory was removed and made opaque to improve performance... in exchange a new window is now being used in the back area of the room with the spiral stair, which looks more pretty and isn't possible to see together it with a lot of other details at once making it much faster.

 - [station_r2620.zip](https://github.com/redeclipse/base/files/14152164/station_r2620.zip)

Added the Corroder weapon, low gravity in the exterior area, various small fixes.

![20230928_185638](https://github.com/redeclipse/base/assets/825418/02b28a79-5b71-4554-8225-743d4a2268ca)
![20231001_053011](https://github.com/redeclipse/base/assets/825418/b43f94f0-f225-4470-9f85-09f983988e78)
![20231001_053040](https://github.com/redeclipse/base/assets/825418/62d26f3f-db1c-49c6-9576-b0f68d9cd5f4)
![20231001_053050](https://github.com/redeclipse/base/assets/825418/d30e1d16-0455-41f6-854c-a8fc350bb2de)
![20231001_053129](https://github.com/redeclipse/base/assets/825418/e14d59ed-3f7d-4aee-aa60-e8caa24de666)
![20231001_053147](https://github.com/redeclipse/base/assets/825418/71967dfc-e2bd-419c-9586-1e24e2c380d9)
![20231001_053159](https://github.com/redeclipse/base/assets/825418/99b764aa-659c-42e2-8f36-c2ac380df652)
![20231001_053208](https://github.com/redeclipse/base/assets/825418/3ee2bd2e-0c93-42ab-b27f-4400584bb78f)
![20231001_053216](https://github.com/redeclipse/base/assets/825418/4170b2e2-9323-4e20-a275-4d5065aa25c4)
![20231001_053233](https://github.com/redeclipse/base/assets/825418/b76bb452-0044-4101-bd6d-8142c9bdac75)
![20231001_053245](https://github.com/redeclipse/base/assets/825418/8f501355-d250-46b9-8d5b-5d79ed315ed5)

**Comment by MirceaKitsune** [comment]
Here are some updated screenshots from today's latest version r2199

![station_20231021_043408](https://github.com/redeclipse/base/assets/825418/a1a75a5a-6b0c-440a-b86f-37403ff37841)
![station_20231021_043431](https://github.com/redeclipse/base/assets/825418/59281686-eab2-44af-9921-fa4045f8eb40)
![station_20231021_043512](https://github.com/redeclipse/base/assets/825418/86766aa9-d158-4e8a-acf6-eb357548001b)
![station_20231021_043547](https://github.com/redeclipse/base/assets/825418/0bc88d8f-4f17-46cb-85eb-d83119694f00)
![station_20231021_043608](https://github.com/redeclipse/base/assets/825418/f82b563f-dcda-4d63-8bec-a49ef0edcc53)
![station_20231021_043635](https://github.com/redeclipse/base/assets/825418/ad92ec78-78d0-4291-9699-5fe34c59a769)
![station_20231021_043648](https://github.com/redeclipse/base/assets/825418/84291157-9b7a-4065-a82c-5b992481ce18)
![station_20231021_043707](https://github.com/redeclipse/base/assets/825418/88607747-ecde-4d20-9ad1-f6bde883a498)
![station_20231021_043735](https://github.com/redeclipse/base/assets/825418/59da1a00-1381-4989-8ca5-d784465bb7bf)
![station_20231021_043756](https://github.com/redeclipse/base/assets/825418/8efb256d-b218-485f-8e95-bef5968b718a)

**Comment by MirceaKitsune** [comment]
Highlights of today's changes in r2341:

![20231021_054814](https://github.com/redeclipse/base/assets/825418/7c5fadd1-5fe2-4ce1-8f34-22010c8a077b)
![20231023_003717](https://github.com/redeclipse/base/assets/825418/302301c9-176e-462b-953f-9e4d5e4fd63f)
![20231023_004417](https://github.com/redeclipse/base/assets/825418/be9377e7-7ad2-467b-bd77-93ccaa97e1b5)
![20231023_004440](https://github.com/redeclipse/base/assets/825418/0ce13cc6-c142-44c4-bb58-bd1657ed64e5)
![20231023_005345](https://github.com/redeclipse/base/assets/825418/17c47afc-1b85-446b-b9cb-bdf11c1f799c)
![20231023_032326](https://github.com/redeclipse/base/assets/825418/2380d688-9878-4346-9d81-9565b026b957)

**Comment by MirceaKitsune** [comment]
Better lighting and detail for the tower in r2439, most else were minor fixes for this rev.

![20231024_005641](https://github.com/redeclipse/base/assets/825418/3e5ed16d-5f5e-45bf-b9bd-13557016ee40)
![20231024_021207](https://github.com/redeclipse/base/assets/825418/201dc967-faba-4f57-8f13-4abddf5b13f6)

**Comment by Jigoku** [comment]
This is really nice. Layout works well! Lot's of interesting paths to cut through 💯 👍 

Two suggestions;

1. The skybox on the second alt variant, it might be moving a bit too fast where it could cause motion sickness. But that's possibly just me waking up not long ago lol. It wouldn't hurt to slow down the spinning a little bit though.


2. Might be worth putting these volumetric lights behind fxlevel 2 or 3:
![Base Profile Screenshot 2023 11 04 - 00 07 04 95](https://github.com/redeclipse/base/assets/1535179/8126f11c-65cb-46c0-b7e6-4a7042dc593b)
![Base Profile Screenshot 2023 11 04 - 00 07 49 08](https://github.com/redeclipse/base/assets/1535179/b7b320ca-7f01-439f-8c28-9b44f8745a1c)


Looks great overall, i particularly like this scene coming out the teleport portal, good use of spark particles too:
![Base Profile Screenshot 2023 11 04 - 00 09 20 38](https://github.com/redeclipse/base/assets/1535179/35aa3c88-7c39-4bed-995f-140fd5d28b61)

**Comment by MirceaKitsune** [comment]
Thanks for the feedback and I'm glad you like the map! Just went ahead and addressed both issues you mentioned in my local version. Still waiting before posting a new update so others can take a look at it in the next days, will fix anything else that comes up and next zip should technically be final.

**Comment by MirceaKitsune** [comment]
[station_r2579.zip](https://github.com/redeclipse/base/files/13401355/station_r2579.zip) is now available, with final improvements following feedback from DC and Jigoku. More information on the specifics in the first post.
